.github/workflow-scripts/run-unit-tests.sh

```diff
@@ -9,15 +9,18 @@
 #   Build Output: status          - Reports build status [failure/success]
 ############################## 
 
+#Setup gradle
 cd dev
 chmod +x gradlew
 
-echo "Running gradle test and testReport tasks.  This will take approx. 30 minutes."
+# global variables
+OUTPUT_FILE=tmp/gradle.log && mkdir -p -- "$(dirname -- "$OUTPUT_FILE")" && touch -- "$OUTPUT_FILE"
+UNIT_DIR=unit-results/ && mkdir -p $UNIT_DIR
```

Comment:
Previously, you had set
UNIT_DIR=`$PWD`/unit-results
now it is:
UNIT_DIR=unit-results/

so the `$PWD` prefix is no longer needed?